src/module-1/cards-list-v1/index-style-card.css

```diff
@@ -0,0 +1,100 @@
+body {
+    background-color: #F9F9F9;
+}
+
+.product-card {
+    display: block;
+    width: 400px;
+    height: 486px;
+    border-radius: 8px;
+    background-color: #FFFFFF;
+    padding-top: 30px;
+    position: relative;
+    font-family: 'Barlow', sans-serif; 
+    box-shadow: 0 10px 20px rgba(0,0,0,0.25), 0 5px 5px rgba(0,0,0,0.22);    
+}
+.imagine {
+    margin: 0 0 35px 0;
+    height: 272px;
+    width: 100%;
+    display: flex;
+    justify-content: center; 
+}
+.rating {
+    margin: 0;
+    height: 100%;
+    width: 70px;
+    background-color: #6F64F8;
+    border-radius: 3px;
+    text-align: center;
+    line-height: 30px;
+    color: white;
+    font-size: 18px;
+    font-weight: 500;  
+}
+.rating-price {
+    height: 30px;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    line-height: 30px;     
+}
+.price, .name, .wishlist {
+    font-size: 18px;
+    font-weight: 600;
+    color: black; 
+}
+.price {
+    margin: auto 25px;        
+}
+.name, .information, .rating {
+    margin-left: 25px;
+}
+.name {
+    margin-bottom: 3px;          
+}
+.information {
+    margin-top: 0;
+    font-size: 16px;
+    font-weight: 500; 
+    color: black; 
+}
+.wishlist, .add {
+    font-family: 'Barlow', sans-serif;   
+    border: none;
+    margin: 0;
+    width: 50%;
+    height: 100%;
+}
+.wishlist {
+    border-radius: 0 0 0 8px;      
+}
+.add {
+    font-weight: 600;
+    font-size: 18px;
+    color: white;
+    background-color: #6F64F8;
+    border-radius: 0 0 8px 0;
+}
+.add:hover, .wishlist:hover, .rating:hover, .imagine:hover{
+    cursor: pointer;
+}
+.buttons {
+    position:absolute;
+    bottom:0;
```

Comment:
Put spaces after `:` in property declarations.

check: https://docs.ckan.org/en/ckan-2.7.3/contributing/css.html